tablepos/TablePos/Logic/TableApp+SPIPayAtTableDelegate.swift

```diff
@@ -64,4 +75,51 @@ extension TableApp: SPIPayAtTableDelegate {
         return response
     }
     
+    func pay(atTableGetOpenTables operatorId: String!) -> SPIGetOpenTablesResponse! {
+        let response: SPIGetOpenTablesResponse = SPIGetOpenTablesResponse()
+        let openTablesArray = NSMutableArray()
```

Comment:
```suggestion
        var openTablesArray: [SPIOpenTablesEntry] = []
```
Couldn't this work? Or does response.openTablesData require an objc-style NSArray?

Reply:
it couldn't work with that. NMutableArray is ok